DashDoc.py

```diff
@@ -80,4 +101,4 @@ def on_cancel(self, name):
         pass
 
     def on_done(self, topic):
-        subprocess.call(["open", "dash-plugin://query=" + quote(topic)])
+        open_dash(topic, False, False)
```

Comment:
keys should be either a `[]` or `None` here, it's confusing to see different types being used.

Reply:
@farcaller, good point! Fixed.